macrel/output.py

```diff
@@ -0,0 +1,129 @@
+readme_output_abundance_mode = """
+--------------------------------------------------------------------------------------------------------------------------
+------------------------------------------          READ  ME        ------------------------------------------------------
+                  Thank you for using Macrel - the (meta)genome AMPs classification and retrieval. 
+                  If you find Macrel useful, please cite:
+                  Santos-Junior, C.D. et al. MACREL: antimicrobial peptide screening in genomes and
+                  metagenomes. The PeerJ 8:e10555 DOI 10.7717/peerj.10555
+--------------------------------------------------------------------------------------------------------------------------
+== This is an explanatory file to the outputs given by Macrel.
+--------------------------------------------------------------------------------------------------------------------------
+Usage example in version 1.0 on the abundance subcommand:
+>>> macrel abundance \
+    -1 example_seqs/R1.fq.gz \
+    --fasta example_seqs/ref.faa.gz \
+    --output out_abundance \
+    --tag example_abundance
+**out_abundance/example_abundance.out.abundance.txt
+    > This file contains a table in the following format:
+        MACREL
+    [1]       [2] 
+    = Field 1. Access of smORFs arbitrarily given by Macrel or the name of the proteins in the reference file
+    = Field 2. Name of the tag given to Macrel or generically "MACREL" and represents the abundance given in reads mapped
+    to the peptide, we recommend to a normalization the user make a pseudo-RPKM calculus as:
+    
+        pseudo-RPKM =  mapped reads * 1000 / total reads * (length in bp)
+"""
+
+readme_output_contigs_mode = """
```

Comment:
Same as above: use markdown.

Or is this `= Field 1` format a standard format that I don't know of.